extensions.json

```diff
@@ -1307,7 +1307,10 @@
     },
     {
         "id": "erlang-ls.erlang-ls",
-        "repository": "https://github.com/erlang-ls/vscode"
+        "repository": "https://github.com/erlang-ls/vscode",
+        "checkout": "master",
+        "prepublish": "npm run compile",
+        "extensionFile": "erlang-ls-*.vsix"
```

Comment:
I don't think that the `extensionFile` attribute here supports wildcards (`*`).

But why do you feel like providing `extensionFile` is needed? (`ovsx` already knows how to package and publish a `.vsix` file without extra guidance)

Reply:
When using `vsce package`, the `*.vsix` file is placed in the root folder of the repository.

After building the `*.vsix` file, where does the `osvx` look for this file?

> But why do you feel like providing extensionFile is needed?

Based on some other pull requests on this repository, I noticed we have to update version number after each new release. So, I thought of reducing technical debt with the use of a wildcard.

If `osvx` can find the published extension, then we don't need extra declarations that will otherwise create confusion.

```suggestion

```

Reply:
Agreed, without the `"extensionFile"` should work fine (actually it already worked once without it: https://open-vsx.org/extension/erlang-ls/erlang-ls)

Reply:
Makes sense.